Readme.md

```diff
@@ -66,21 +66,3 @@ Support
 -------
 
 SimpleITK provides access to most of the ITK components, but not all. If you are looking for something specific and can't find it, open an issue on GitHub or ask for support on the [forum](https://discourse.itk.org).
-
-Report an Issue
----------------
```

Comment:
For less GitHub savvy folks, possibly modify to:

Report an Issue, Request a Feature
-----------------------------------
Please use the [issue tracker](https://github.com/SimpleITK/SimpleITK/issues) to report issues or request features.

Reply:
done